scheds/kernel-examples/scx_central.bpf.c

```diff
@@ -142,6 +142,13 @@ static bool dispatch_to_cpu(s32 cpu)
 	s32 pid;
 
 	bpf_repeat(BPF_MAX_LOOPS) {
+		/* We might run out of dispatch buffer slots if we continue dispatching
```

Comment:
Maybe use fully winged comment style to be consistent with other comments?

Reply:
Ack, will do.